TrafficCapture/captureKafkaOffloader/src/main/java/org/opensearch/migrations/trafficcapture/kafkaoffloader/KafkaCaptureFactory.java

```diff
@@ -8,16 +8,22 @@
 import org.opensearch.migrations.trafficcapture.IChannelConnectionCaptureSerializer;
 import org.opensearch.migrations.trafficcapture.IConnectionCaptureFactory;
 import org.opensearch.migrations.trafficcapture.StreamChannelConnectionCaptureSerializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
 
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.util.Arrays;
 import java.util.WeakHashMap;
 import java.util.concurrent.CompletableFuture;
 
+
 @Slf4j
 public class KafkaCaptureFactory implements IConnectionCaptureFactory {
 
+    private static final Logger metricsLogger = LoggerFactory.getLogger("MetricsLogger");
```

Comment:
Please canonicalize this within code.  A MetricsLogger class can return WrappedLogger objects that return LoggingEventBuilder objects for errorEvents, successEvents, and traceEvents (if we ever think that those are useful).  Then you can document what the log levels (trace, debug, info, warn, error) mean.  You can also make sure that the loggers are all named something consistent.  "MetricLogger.KafkaCaptureFactory" would make sense for MetricLogger.getLogger("KafkaCaptureFactory").